src/status_im/ui/screens/discover/components/views.cljs

```diff
@@ -55,29 +55,31 @@
     (let [{:keys [name photo-path whisper-id message-id status]} message
           {account-photo-path :photo-path
            account-address    :public-key
-           account-name       :name} current-account
-          me?        (= account-address whisper-id)
-          item-style (get-in platform/platform-specific [:component-styles :discover :item])]
+           account-name       :name}                             current-account
```

Comment:
We usually don't align in `let` blocks. @rasom WDYT?

Reply:
No, we do, but for instance, intellij treats desctructuring in `let` as special case and aligns only symbol bindings. So it will look like:
```clojure
(let [{:keys [name photo-path whisper-id message-id status]} message
      {account-photo-path :photo-path
       account-address    :public-key
       account-name       :name} current-account
      me?        (= account-address whisper-id)
      item-style (get-in platform/platform-specific [:component-styles :discover :item])]
  ...)
```

Reply:
Ok might be worth documenting this

Reply:
I think we decided to align in lets and maps, and then we should go with it all-in, not with what <specific IDE> decided to do, imo. I.e. it should be:

```
(let [{:keys [name photo-path whisper-id message-id status]} message
      {account-photo-path :photo-path
       account-address    :public-key
       account-name       :name} current-account
      me?                                      (= account-address whisper-id)
      item-style                            (get-in platform/platform-specific [:component-styles :discover :item])]
  ...)
```

Reply:
@jeluard  Wiki Friday style guide doc maybe?